### ISSUE TITLE
Strange behavior when combining Direct and Rule property changes

### ISSUE THREAD

**Issue by fkgruber** [tmuetze/Bioconductor_RCy3_the_new_RCytoscape#40]
I noticed that when I set some of the properties using something like `setNodeColorDirect` and then later in the code set a rule like `'setNodeBorderColorRule` the results are unexpected and wrong.

I have crated a gist and hopefully you can replicate

[https://gist.github.com/fkgruber/bfab532519f00fb11a259aeb54911d80](url)

For this code sometimes I get one blue node, 2 white nodes and no borders.

**Comment by AlexanderPico** [comment]
We are in the midst of overhauling RCy3, including new approaches for defining visual property mappings. I know this was a year ago... but if you still have a use case in this regard, please post it to the new repo and I'll make sure it is covered. (the gist link above no longer works).

New repo: https://github.com/cytoscape/RCy3

**Comment by fkgruber** [comment]
thanks
I will take a look at the new version

**Comment by tmuetze** [comment]
@fkgruber Yes, this is a known bug: https://github.com/cytoscape/RCy3/blob/master/NEWS
